app/server/utils/credit_transfer.py

```diff
@@ -43,68 +43,77 @@ def calculate_transfer_stats(total_time_series=False, start_date=None, end_date=
         dateFilter.append(CreditTransfer.created >= start_date)
         dateFilter.append(CreditTransfer.created <= end_date)
 
-    filters = [CreditTransfer.transfer_type == TransferTypeEnum.PAYMENT, CreditTransfer.transfer_subtype == TransferSubTypeEnum.DISBURSEMENT]
+    disbursement_filters = [
```

Comment:
continual overwriting of the `filters` var (which isn't really a var but more of a const) felt a bit iffy and prone to unexpected errors, and this allows for re-use over multiple queries.